packages/common/src/ConfiguFile.ts

```diff
@@ -5,13 +5,16 @@ import { findUp } from 'find-up';
 import { ConfigSchema, ConfigStore, Expression, JsonSchema, JsonSchemaType } from '@configu/sdk';
 import FastGlob from 'fast-glob';
 import _ from 'lodash';
-import { readFile, parseJSON, parseYAML, mergeSchemas } from './utils';
```

Comment:
Cleanup of a bug I introduced on a previous PR (surfaced as a merge conflict when rebasing)

Reply:
Good catch